test/python3/libdnf5/tutorial/session/create_base.py

```diff
@@ -11,6 +11,16 @@
 # configuration from a different location.
 base.load_config_from_file()
 
+# Optionally you can set and get vars
+# vars = base.get_vars().get()
+# vars.set('releasever', '33')
+#
+# Once a variable is set it can be accessed by the getter method
+# vars.get('releasever')
```

Comment:
I believe that currently this approach does not work. `vars.get('releasever')` returns something like a pair, but a structure is unknown in python, therefore you cannot use it at all (bug) . What about to describe - `get_priority` and `get_value`? Hope that get priority works

Reply:
I am sorry - my previous comment was incorrect - get_priority() is not available method for Vars

Get() returns Vars structure that contains
```
    struct Variable {
        std::string value;
        Priority priority;
    };
```

Reply:
Because get() approach is not working - ok working but not useful - I would recommend to remove example with get() and add todo - provide a way how to get option  priority. What do you think?

Reply:
@inknos May I ask you what's yours plans?